pythonforandroid/bootstraps/webview/build/src/main/java/org/kivy/android/PythonActivity.java

```diff
@@ -87,13 +95,20 @@ protected void onCreate(Bundle savedInstanceState) {
         resourceManager = new ResourceManager(this);
 
         Log.v(TAG, "Ready to unpack");
-        unpackData("private", getFilesDir());
+        File app_root_file = new File(getAppRoot());
+        unpackData("private", app_root_file);
 
-        this.mActivity = this;
+        Log.v(TAG, "About to do super onCreate");
```

Comment:
this looks familiar, isn't this change on master already? https://github.com/kivy/python-for-android/blob/master/pythonforandroid/bootstraps/sdl2/build/src/main/java/org/kivy/android/PythonActivity.java#L66 
is this pull request rebased, or is master merged into it somehow?

it would be quite good if we could see what is actually new, so if the history could be cleaned up that would be very useful